main/Fluids.F90

```diff
@@ -228,6 +231,9 @@ SUBROUTINE FLUIDS()
          & default=1)
     call get_option("/timestepping/nonlinear_iterations/tolerance", &
          & nonlinear_iteration_tolerance, default=0.0)
+
+    !Initialise positions of particles before adapt_at_first_timestep
```

Comment:
Sorry to be pedantic: but could you add a space after the comment marker (and in the lines below)